Source/render/GroupRenderer.swift

```diff
@@ -51,14 +51,14 @@ class GroupRenderer: NodeRenderer {
         return nil
     }
 
-    override func doFindAllNodesAt(location: CGPoint, ctx: CGContext) -> [Node]? {
+    override func doFindAllNodesAt(location: CGPoint, ctx: CGContext) -> [NodeTouch]? {
```

Comment:
It's too complicated. Let's use following class:
```swift
class NodePath {
  let node: Node
  let location: CGPoint
  let parent: NodePath?
}
```
Also do we still need `doFindNodeAt` method? Can we have only one method to find node by location?